bundles/org.openhab.binding.modbus.sbc/README.md

```diff
@@ -0,0 +1,75 @@
+# Modbus Saia Burgess Controls Binding
+
+This binding interfaces the energy meter series ALD1 by Saia Burgess Controls (SBC) via Modbus.
+
+## Supported Things
+
+The following Things are supported:
+
+- `ald1Unidirectional`: 1-phase 32A one-way energy meter ALD1D5FD00A3A00
+- `ald1Bidirectional`:  1-phase 32A two-way energy meter ALD1B5FD00A3A00
+
+## Discovery
+
+This binding does not support discovery.
+
+## Thing Configuration
+
+The following configuration parameters apply to `ald1Unidirectional` and `ald1Bidirectional`.
+
+| Name          | Description                              | Type    | Required |
+|---------------|------------------------------------------|---------|----------|
+| address       | Modbus slave address of the energy meter | Integer | yes      |
```

Comment:
This is already specified in the parent serial thing, I think it makes sense to use it from there and remove this parameter.

Reply:
Great hint! To retrieve the slave address I modified the BaseModbusThingHandler. This was missing in the current implementation.